sky/services/editing/ios/keyboard_impl.mm

```diff
@@ -57,13 +58,15 @@ - (void)show {
   NSAssert([UIApplication sharedApplication].keyWindow != nullptr,
            @"The application must have a key window since the keyboard client "
            @"must be part of the responder chain to function");
-  [[UIApplication sharedApplication].keyWindow addSubview:self];
+  if (!self.superview)
+    [[UIApplication sharedApplication].keyWindow addSubview:self];
   [self becomeFirstResponder];
 }
 
 - (void)hide {
   [self resignFirstResponder];
-  [self removeFromSuperview];
+  if (self.superview)
```

Comment:
This check is redundant. Ditto for the `addSubview:` call above. Adding a view as a subview removes it from its previous parent.